3478/1/top_gems.rb

```diff
@@ -0,0 +1,6 @@
+require 'dotenv'
```

Comment:
I like that you have mastered this wonderful gem and git ignor, but it this case, throw it to me in a personal, so that I can run your code.

Reply:
How can I share my token? I can't upload it to GitHub. GitHub checks if this token was uploaded and disables it.